testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/miniapplist/MiniAppListActivity.kt

```diff
@@ -39,6 +39,7 @@ class MiniAppListActivity : MenuBaseActivity() {
         val intent = Intent(this, SettingsMenuActivity::class.java)
         intent.putExtra(MENU_SCREEN_NAME, MINI_APP_LIST_ACTIVITY)
         startActivity(intent)
+        finish()
```

Comment:
Shouldn't you call a startActivityForResult and check if the setting screen has not been canceled ?

Reply:
That's another solution. `startActivityForResult` can be used when we will have a spec to carry data.
Since now, it's a matter of reloading the entire screen after saved settings, we can go with current implementation.

Reply:
But it seems using `finish` will cause another issue: tapping the back button from the settings screen will close the App because the previous screen has been finished. I'm not really sure what the best solution is. You could use `startActivityForResult` or find some other fix.

Reply:
Yeah, I got it. I have updated this using `startActivityForResult`, for now I am using `RESULT_CANCELED` instead of `RESULT_OK` as I don't need to carry any data while returning back from settings.